Makefile

```diff
@@ -53,6 +53,7 @@ inst: ## go install tools
 	go install github.com/client9/misspell/cmd/misspell@v0.3.4
 	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
 	go install github.com/goreleaser/goreleaser@v1.18.2
+	go install github.com/cpuguy83/go-md2man/v2
```

Comment:
I get an error when installing this module:
```
go install github.com/cpuguy83/go-md2man/v2
missing go.sum entry for module providing package github.com/cpuguy83/go-md2man/v2; to add:
	go mod download github.com/cpuguy83/go-md2man/v2
```
We don't need this module in `go.mod` so it doesn't make sense to run the above `go mod download` command. This can be fixed by appending the version onto the end of the module path, e.g:

```
go install github.com/cpuguy83/go-md2man/v2@latest
```
or whichever version you choose to bind to.

Reply:
This looks good, resolving.